libs/zedUpload/datastore.go

```diff
@@ -87,10 +88,16 @@ type DronaCtx struct {
 func (ctx *DronaCtx) ListenAndServe() {
 	for {
 		select {
-		case req := <-ctx.reqChan:
-			_ = ctx.handleRequest(req)
-
+		case req, ok := <-ctx.reqChan:
+			if ok {
+				logrus.Infof("XXX ListenAndServe got request")
```

Comment:
Do you want to keep the `XXX` prefix in the logs? What about something specific that we could use to filter these (and only these) logs in Kibana, like `Downloader channel:` or something similar.

Reply:
My plan is to remove the XXX and downgrade some of the logs but I first want to see if I can get some of this close of the cancel channel to happen during testing.

Reply:
Done